### PR TITLE
layering the path for Parts based crafting and new crafted weapon. [sprites=placeholders]

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
adds:

**Handmade bolt action rifle (.30)**. Cheap rifle that only requires 2 gun parts, 5 plastic and 10 steel to made. Have a magazine of 5, and by any means is a worse version of serbian one. 
  image  this was tough
**Handmade revolver (.40)**. Requires 2 gun parts to be made, as well as 10 plastic and 15 steel. Can hold 5 shells, a worse version of any revolver, yet is capable of killing someone with full chamber unloaded. 
image 
**Handmade Pistol (.40)**. Already exist. Does not require gun parts, only 5 untis of steel and 5 units of wood. Reworked for .40 magnum.  _**[icon already in game]**_
**Slide barrel shotgun (shells)**. Requires 3 gun parts to be made, as well as 10 wood and 20 steel. Can hold only 1 shell, but can be tactically reloaded, by pressing an ammo pile on it without ejecting. It’s also can be used in one hand with small debuf. For all other purpose, again, have worse stats. 
image 
 (and a special loading animation)
image   
/of course it only runs once IC
**Motherfucker (.35)**. Requires 5 gun parts to be made, as well as 15 wood and 20 steel. It is a 6 barrel, pump action carbine, that can only fire all 6 barrels at once, requiring to pump after each barrage. It have a top mounted non detachable magazine with capacity of 54, that must be loaded by hand. Non existent accuracy and hellish recoil should balance the damage out. If not - lower the damage. 
image   
**Makeshift laser carbine**. Requires 4 gun parts to be done, as well as 20 steel, 15 plastic and a bunch of microlasers. While, again, being a worse version of Lightfall, the stat gap is not really huge, and it’s capable of dealing same amount of damage, just for greater charge cost and can’t fire too often.  **_[unused icon already in the code for "energy gun"]_**
**Makeshift grenade launcher**. Requires 2 gun parts to be made, as well as 10 wood and 20 steel. Can hold one grenade in a pipe. Basically, your own China Lake.
image   
Armor already have some makeshift variants of clothes, but let’s include few new:

**Makeshift armored void suit**. Requires 3 armor parts, 10 plasteel, 10 glass, 10 duct tape, 20 plastic and any jumpsuit to be made. Comes with helmet. Revenge from a junkyard, slow, have some basic armor, and can protect from space or viruses.
BARON SPRITES  
image    
**Scavenger armor**.  Requires 5 armor parts, 5 plasteel, 10 glass, 10 duct tape, 10 plastic. A long brown coat with plasteel armor on torso, lined with pockets for storage, **with armor that can be compared to basic IH.**
image   

_**gun parts. and armor parts. take at the moment 15plasteel, and 15plastic to make respectively, BUT unlike the Document idea they are not unique at the moment, neither have special stats**_

## Why It's Good For The Game
will layer some of the path for loot rework

## Changelog
:cl: Fernandos33, with RedBaron Sprites
add: Handmade bolt action rifle (.30)
add: Handmade revolver (.40)
add: Handmade Pistol (.40)
add: Slide barrel shotgun (shells)
add: Motherfucker (.35)
add: Makeshift laser carbine sprited by
add: Makeshift grenade launcher
add: Makeshift armored void suit
add: Scavenger armor
/:cl:

### CODE REVIEW

**Comment by fernandos13** [comment]
pretty sure people will be pleased, that i closed my prs. i was after all annoying

**Comment by fernandos13** [comment]
i'm annoying. and i did my annoying part of today

File: code/modules/clothing/suits/miscellaneous.dm
Comment (AMM101902) [inline code/modules/clothing/suits/miscellaneous.dm:236]: Not even the  basic ih armor vests get arm protection, that's probably too good for joonk vest.

File: code/modules/projectiles/guns/energy/laser.dm
Comment (AMM101902) [inline code/modules/projectiles/guns/energy/laser.dm:152]: This is better than the Cog, a "real" gun. You might want to bump the charge cost up or reduce the damage. For reference the Cog has 1 dmg mult and 100 charge cost

File: code/modules/projectiles/guns/projectile/boltgun.dm
Comment (AMM101902) [inline code/modules/projectiles/guns/projectile/boltgun.dm:109]: you have recoil buildup in twice, just delete this one.

File: code/modules/projectiles/guns/projectile/shotgun/slidebarrelshotgun.dm
Comment (AMM101902) [inline code/modules/projectiles/guns/projectile/shotgun/slidebarrelshotgun.dm:17]: This is probably too low. To be fair the sawn-off also has its recoil set too low.

**Review by AMM101902 — COMMENTED** [review]
It looks like you have the Motherfucker in twice, once under shotgun, and once under automatics. Is that intentional?

File: code/modules/clothing/suits/miscellaneous.dm
Comment (fernandos13) [reply code/modules/clothing/suits/miscellaneous.dm:236]: wow i didn't knew that. """wow IH vest sucks...""" i thought it was one of the best...

Comment (fernandos13) [reply code/modules/clothing/suits/miscellaneous.dm:236]: overlooked it. well changes being made

File: code/modules/projectiles/guns/energy/laser.dm
Comment (fernandos13) [reply code/modules/projectiles/guns/energy/laser.dm:152]: well making it worst in both ways

File: code/modules/projectiles/guns/projectile/shotgun/slidebarrelshotgun.dm
Comment (fernandos13) [reply code/modules/projectiles/guns/projectile/shotgun/slidebarrelshotgun.dm:17]: oh, wow.. most have 14, and the pump got.. 1.2 that is indeed.. akward.. and low.. well making the slide 16

**Comment by fernandos13** [comment]
> It looks like you have the Motherfucker in twice, once under shotgun, and once under automatics. Is that intentional?

definetly not. that file shouldn't even be there... lemme delete it and upload the changes RN, thanks for your time

**Comment by soslow4u** [comment]
pls, `image` for images

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

It's a good thing to have more makeshift weapons. The current makeshift weapons we have (chopper and handmade pistol) are never used because of how bad they are in every respect. Weapons like these are unique and add to the experience of junking without praying to RNGsus for any gun or voidsuit.

Still needs code review from a maintainer.

**Comment by soslow4u** [comment]
merge conflict

**Review by genessee596 — CHANGES_REQUESTED** [review]
Hey, actually - I know I approved this before, but I actually need you to change a couple things to be in accordance to the main design doc.

So the biggest thing here is making it so gun/armor parts can spawn in junk piles in place of actual guns/armor instead of just being craftable. The whole point of having these junk weapons is because finding actual weapons should be difficult to do unless you're extremely lucky.

**Review by Hopekz — CHANGES_REQUESTED** [review]
This PR is getting stale.
We need this part to work with the loot rework since those are listed as the prerequisite for this part in the design doc

**Comment by genessee596** [comment]
Hey, just a headsup. If you don't make a response to our requests by tomorrow morning, I'm going to assume you're not actually working on this anymore and I'm going to close it.

Thanks.

**Pull request closed without merging**
